operations/mimir-mixin-tests/test-custom-labels-asserts.sh

```diff
@@ -20,7 +20,7 @@ assert_failed() {
 }
 
 # In this test we customize some labels. We expect to not find any reference to default labels.
-FORBIDDEN_LABELS="cluster instance pod"
+FORBIDDEN_LABELS="cluster instance node"
```

Comment:
This assertion shouldn't change. `pod` is the default value of `per_instance_label`, which is what we want to assert here.